app/shared/viewmodel.js

```diff
@@ -54,7 +54,7 @@ function ContainerViewModel(data) {
   this.Status = data.Status;
   this.Names = data.Names;
   // Unavailable in Docker < 1.10
-  if (data.NetworkSettings) {
+  if (data.NetworkSettings && Object.keys(data.NetworkSettings.Networks).length > 0) {
```

Comment:
`&& !_.isEmpty(data.NetworkSettings.Networks` should be used as a check. It has already been merged in the develop branch.